### ISSUE TITLE
Change UI font command doesn't work

### ISSUE THREAD

**Issue by skrewde** [everythingsucksass/ui-font-changer#1]
As indicated in the title, the command to change VS Code's UI font doesn't work and throws out this error:
![image](https://github.com/everythingsucksass/ui-font-changer/assets/94148640/ae16451d-64b3-46fa-87f5-0e3bae1d74f7)

It's likely that the extension was broken by an edit. Also, editing the workbench CSS and JS work just fine.

**Comment by everythingsucksass** [comment]
The extension creates a backup of the orginal files, looks like it lacked permission to create them.

I think this is related to your Windows settings.

**Comment by skrewde** [comment]
Any idea what setting might've caused that even while running with Administrator privileges?

Also, when I said edit in my last comment, I meant update.

**Comment by KunalTanwar** [comment]
I think this is the problem is related to the specified file name i.e. `workbench.desktop.min.css.backup` it should be `workbench.desktop.backup.min.css`. Maybe this will fix the problem.

**Comment by everythingsucksass** [comment]
> Also, when I said edit in my last comment, I meant update.

So this error does not occur when you change the font for the first time, but occurs when you try to change it again?

**Comment by everythingsucksass** [comment]
> I think this is the problem is related to the specified file name i.e. `workbench.desktop.min.css.backup` it should be `workbench.desktop.backup.min.css`. Maybe this will fix the problem.

Thanks, Will look into it.

**Comment by skrewde** [comment]
No, that was my first time trying out the extension. I just wondered if a VS Code update (after release) broke the extension.

**Comment by everythingsucksass** [comment]
@skrewde, I just checked if the updates broke anything with a fresh install and everything works fine.

@KunalTanwar, The file names don't seem to be the problem.

After some Google searches for the error in your screenshot, this seems to be related to Node.js and folder permission.

Honestly, this is way over my head.

**Comment by skrewde** [comment]
Understandable, I don't use Node/Typescript much either so it might take a while to debug.

I'm curious though, do you have the [workspace trust](https://code.visualstudio.com/docs/editor/workspace-trust) settings enabled in VS Code?

**Comment by everythingsucksass** [comment]
Yes, it's enabled.

**Comment by KunalTanwar** [comment]
@skrewde Are you administrator I mean you aren't using guest account right?

**Comment by skrewde** [comment]
No, I always run as Admin
